updateHostsWindows.bat

```diff
@@ -20,11 +20,25 @@ if not exist "%WINDIR%\System32\drivers\etc\hosts.skel" (
 	COPY %WINDIR%\System32\drivers\etc\hosts %WINDIR%\System32\drivers\etc\hosts.skel
 )
 
+:: Make sure we are in the correct working directory
+rem Path info from:
+rem https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call#batch-parameters
+cd %~dp0

```

Comment:
Why `cd` if we assume that we already are in the correct dirctory?

Reply:
Because it's not a valid assumption. At least on Windows 10, whatever build I'm on, batch files run as administrator have a working directory of `%windir%` and they seem ignore the "Start in" option in Properties. (Yeah, this caused more than a few bad words to figure out.)

Reply:
Strange :thinking: Someone will need to confirm this as we never needed that before...

Reply:
It's been a long time since I was on Windows before, but it's not the behavior I remember either. That said, this whole "Run as administrator" is also somewhat new to me.